src/library_openal.js

```diff
@@ -1008,6 +1011,42 @@ var LibraryOpenAL = {
     }
   },
 
+  alGetSourceiv__deps: ['alGetSourcei'],
+  alGetSourceiv: function(source, param, values) {
+    if (!AL.currentContext) {
+#if OPENAL_DEBUG
+      console.error("alGetSourceiv called without a valid context");
+#endif
+      return;
+    }
+    var src = AL.currentContext.src[source];
+    if (!src) {
+#if OPENAL_DEBUG
+      console.error("alGetSourceiv called with an invalid source");
+#endif
+      AL.currentContext.err = 0xA001 /* AL_INVALID_NAME */;
+      return;
+    }
+    switch (param) {
+    case 0x202 /* AL_SOURCE_RELATIVE */:
+    case 0x1001 /* AL_CONE_INNER_ANGLE */:
+    case 0x1002 /* AL_CONE_OUTER_ANGLE */:
+    case 0x1007 /* AL_LOOPING */:
+    case 0x1009 /* AL_BUFFER */:
+    case 0x1010 /* AL_SOURCE_STATE */:
+    case 0x1015 /* AL_BUFFERS_QUEUED */:
+    case 0x1016 /* AL_BUFFERS_PROCESSED */:
+      this._alGetSourcei(source, param, values);
```

Comment:
We have not generally used the form `this._alGetSourcei(...)` to call dependent functions, just `_alGetSourcei(...)` is enough. Is there a reason to prefer using `this.`?